src/Domain.Schema/Request/Control/WriteValues/PrimitiveTypes/Base/WriteRequestPrimitiveArray.cs

```diff
@@ -15,5 +15,10 @@ public WriteRequestPrimitiveArray(IEnumerable<TType> items)
 
         Array IPrimitiveArray.ToArray()
             => this.ToArray();
+
+        public override string ToString()
```

Comment:
WriteProvider.cs

When we log a debug message

  this.Logger.Debug($"Executed {this.Commands.FirstOrDefault()?.GetType().Name} on NodeId: [{nodeId}] with value: [{command.Value}]");

For arrays the class name is logged instead of the values